packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -181,5 +181,119 @@
             "throw new ${1:Exception}($0)"
         ],
         "description": "Throw a new exception"
+    },
+    "list": {
+        "prefix": "list<",
+        "body": [
+            "list<${1:object}> ${2:listName} = new list<$1>();"
+        ],
+        "description": "list constructor"
+    },
+    "map": {
+        "prefix": "map<",
+        "body": [
+            "map<${1:object}, ${2:object}> ${3:mapName} = new map<$1, $2>();"
+        ],
+        "description": "map constructor"
+    },
+    "set": {
+        "prefix": "set<",
+        "body": [
+            "set<${1:object}> ${2:setName} = new set<$1>();"
+        ],
+        "description": "set constructor"
+    },
+    "@AuraEnabled": {
+        "prefix": "@AuraEnabled",
+        "body": [
+            "@AuraEnabled",
+            "${1|public,global|} static ${2:string} ${3:methodName}(){",
+            "\t$4",
+            "}"
+        ],
+        "description": "Aura Enabled Apex Method"
+    },
+    "userCreation": {
```

Comment:
I'm not sure how common this snippet is. I would put this as https://code.visualstudio.com/docs/editor/userdefinedsnippets and not put this into the general one.

Reply:
I made the profile name a variable so that it's more useful for customers (testing stuff as one of their own profile).

Testing with runAs as a non-admin is considered a best practice...
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_testing_example.htm
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_testing_best_practices.htm

...and assuming that a user exists was fine for sandboxes but won't work in scratch orgs.  @vazexqi do you see an alternative to this snippet?

Reply:
Let's remove this snippet for now. I am iffy on it and I would prefer to not have too many snippets first so that we can keep this lean. For your own purposes, you can set it up using https://code.visualstudio.com/docs/editor/userdefinedsnippets.

Reply:
Removed it.  These all came from my userdefinedsnippets file